### PR TITLE
HIVE-26100: Preparing for 4.0.0-alpha-2 development

### PR DESCRIPTION
**Pull request by pvary**
### What changes were proposed in this pull request?
New pom.xml versions, and new db files

### Why are the changes needed?
So the next release development could be started

### Does this PR introduce _any_ user-facing change?
New db version

### How was this patch tested?
Unit tests

### CODE REVIEW

File: standalone-metastore/metastore-server/src/main/sql/mysql/upgrade-4.0.0-alpha-1-to-4.0.0-alpha-2.mysql.sql [hunk 1]
```diff
@@ -0,0 +1,240 @@
+SELECT 'Upgrading MetaStore schema from 4.0.0-alpha-1 to 4.0.0-alpha-2' AS MESSAGE;
```
Comment (zabetak) [inline standalone-metastore/metastore-server/src/main/sql/mysql/upgrade-4.0.0-alpha-1-to-4.0.0-alpha-2.mysql.sql:1]: Shouldn't this MySQL upgrade script be mostly empty (except version update) at the moment?

File: standalone-metastore/metastore-server/src/main/sql/postgres/upgrade-4.0.0-alpha-1-to-4.0.0-alpha-2.postgres.sql [hunk 2]
```diff
@@ -0,0 +1,374 @@
+SELECT 'Upgrading MetaStore schema from 4.0.0-alpha-1 to 4.0.0-alpha-2';
```
Comment (zabetak) [inline standalone-metastore/metastore-server/src/main/sql/postgres/upgrade-4.0.0-alpha-1-to-4.0.0-alpha-2.postgres.sql:1]: Shouldn't this Postgres upgrade script be mostly empty (except version update) at the moment?

File: metastore/scripts/upgrade/hive/upgrade-4.0.0-alpha-1-to-4.0.0-alpha-2.hive.sql [hunk 3]
```diff
@@ -0,0 +1,887 @@
+SELECT 'Upgrading MetaStore schema from 3.1.0 to 4.0.0-alpha-1';
```
Comment (zabetak) [inline metastore/scripts/upgrade/hive/upgrade-4.0.0-alpha-1-to-4.0.0-alpha-2.hive.sql:1]: Shouldn't this Hive upgrade script be mostly empty (except version update) at the moment?

Also the query should be the following I think:
`SELECT 'Upgrading MetaStore schema from 4.0.0-alpha-1 to 4.0.0-alpha-2';`

File: metastore/scripts/upgrade/hive/upgrade-4.0.0-alpha-1-to-4.0.0-alpha-2.hive.sql [hunk 4]
```diff
@@ -0,0 +1,887 @@
...
+                        JOIN `sys`.`DBS` D ON (C.`C_DATABASE` = D.`NAME`)
+                        LEFT JOIN `sys`.`TBL_PRIVS` P ON (T.`TBL_ID` = P.`TBL_ID`)
+WHERE
+  (NOT restrict_information_schema() OR P.`TBL_ID` IS NOT NULL
+  AND (P.`PRINCIPAL_NAME`=current_user() AND P.`PRINCIPAL_TYPE`='USER'
+    OR ((array_contains(current_groups(), P.`PRINCIPAL_NAME`) OR P.`PRINCIPAL_NAME` = 'public') AND P.`PRINCIPAL_TYPE`='GROUP'))
+  AND P.`TBL_PRIV`='SELECT' AND P.`AUTHORIZER`=current_authorizer());
+
+
+SELECT 'Finished upgrading MetaStore schema from 3.1.0 to 4.0.0-alpha-1';
```
Comment (zabetak) [inline metastore/scripts/upgrade/hive/upgrade-4.0.0-alpha-1-to-4.0.0-alpha-2.hive.sql:887]:
```suggestion
SELECT 'Finished upgrading MetaStore schema from 4.0.0-alpha-1 to 4.0.0-alpha-2';
```

**Review by zabetak — COMMENTED** [review]
Left a few comments around the metastore upgrade scripts.

File: standalone-metastore/metastore-server/src/main/sql/mysql/upgrade-4.0.0-alpha-1-to-4.0.0-alpha-2.mysql.sql (hunk 1 above)
Comment (pvary) [reply standalone-metastore/metastore-server/src/main/sql/mysql/upgrade-4.0.0-alpha-1-to-4.0.0-alpha-2.mysql.sql:1]: Thanks for catching this.
Fixed

File: standalone-metastore/metastore-server/src/main/sql/postgres/upgrade-4.0.0-alpha-1-to-4.0.0-alpha-2.postgres.sql (hunk 2 above)
Comment (pvary) [reply standalone-metastore/metastore-server/src/main/sql/postgres/upgrade-4.0.0-alpha-1-to-4.0.0-alpha-2.postgres.sql:1]: Thanks for catching this.
Fixed

File: metastore/scripts/upgrade/hive/upgrade-4.0.0-alpha-1-to-4.0.0-alpha-2.hive.sql (hunk 4 above)
Comment (pvary) [reply metastore/scripts/upgrade/hive/upgrade-4.0.0-alpha-1-to-4.0.0-alpha-2.hive.sql]: Thanks for catching this.
Fixed

**Comment by pvary** [comment]
@zabetak: Fixed them. Thanks for the review!

**Review by zabetak — APPROVED** [review]
LGTM! Thanks for taking this one Peter.

**Pull request merged**